Console/Command/NostoAccountRemoveCommand.php

```diff
@@ -0,0 +1,217 @@
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @author Nosto Solutions Ltd <contact@nosto.com>
+ * @copyright 2017 Nosto Solutions Ltd
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD 3-Clause
+ *
+ */
+
+namespace Nosto\Tagging\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Nosto\Tagging\Helper\Account as NostoAccountHelper;
+use Nosto\Tagging\Helper\Scope as NostoScopeHelper;
+use Nosto\Object\Signup\Account as NostoSignupAccount;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Store\Model\Store;
+use Magento\Framework\App\Config\Storage\WriterInterface;
+
+class NostoAccountRemoveCommand extends Command
+{
+    /**
+     * Path to store config nosto account name.
+     */
+    const XML_PATH_ACCOUNT = 'nosto_tagging/settings/account';
+
+    /**
+     * Path to store config nosto account tokens.
+     */
+    const XML_PATH_TOKENS = 'nosto_tagging/settings/tokens';
+    const NOSTO_ACCOUNT_ID = 'account-id';
+    const TOKEN_SUFFIX = '_token';
+    const SCOPE_CODE = 'scope-code';
+
+    /*
+    * @var NostoAccountHelper
+    */
+    private $nostoAccountHelper;
+
+    /**
+     * @var NostoHelperScope
+     */
+    private $nostoScopeHelper;
+
+    /**
+     * @var bool
+     */
+    private $isInteractive = true;
+
+    /**
+     * @var WriterInterface
+     */
+    private $config;
+
+    /**
+     * NostoAccountRemoveCommand constructor.
+     * @param NostoAccountHelper $nostoAccountHelper
+     * @param NostoScopeHelper $nostoScopeHelper
+     * @param WriterInterface $appConfig
+     */
+    public function __construct(
+        NostoAccountHelper $nostoAccountHelper,
+        NostoScopeHelper $nostoScopeHelper,
+        WriterInterface $appConfig
+    )
+    {
+        $this->nostoAccountHelper = $nostoAccountHelper;
+        $this->nostoScopeHelper  = $nostoScopeHelper;
+        $this->config = $appConfig;
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure()
+    {
+        $this->setName('nosto:account:remove')
+            ->setDescription('Remove Nosto Account Via CLI')
+            ->addOption(
```

Comment:
We don't really care about the account name here. It's enough if the store cope is set.